package.json

```diff
@@ -58,6 +59,7 @@
     "eslint-config-prettier": "^3.3.0",
     "gh-pages": "^2.0.1",
     "husky": "^1.2.1",
+    "jest": "^23.6.0",
```

Comment:
@EdGrLab why did you left jest in declarations?!
It's included by default in **CRA** application